### ISSUE TITLE
Memory leak in ASNetworkImageNode

### ISSUE THREAD

**Issue by codeOfRobin** [facebookarchive/AsyncDisplayKit#3059]
There appear to be memory leaks in instances of `NSURLRequest`stemming from `ASBasicImageDownnloader`

![screen shot 2017-02-22 at 12 41 22 pm](https://cloud.githubusercontent.com/assets/5009041/23200775/40ac7c18-f8fc-11e6-8898-f18353e6e688.png)
![screen shot 2017-02-22 at 12 41 19 pm](https://cloud.githubusercontent.com/assets/5009041/23200778/421e8ea6-f8fc-11e6-9ba7-d8dca2e2fdb2.png)

**Comment by maicki** [comment]
@codeOfRobin Hey! I just looked briefly into it, but could you ran your tests / Instrument build again the `ASDKgram` example and provide a bit more details from the Instruments. Currently it's hard to follow and reproduce. Thanks!

**Comment by garrettmoon** [comment]
Going to close this out since there hasn't been a reply.
